RallyHereIntegration/Source/RallyHereIntegration/Private/RH_LocalPlayerSessionSubsystem.cpp

```diff
@@ -772,8 +786,17 @@ void URH_LocalPlayerSessionSubsystem::OnPlatformActivityActivation(const FUnique
 
 	// we have received a notification that the user accepted an invitation from the system.  We need to attempt to join that session (at which point we will resynchronize with it via the RHSession)
 
-	// we need to join the session
-	URH_PlatformSessionSyncer::JoinRHSessionByPlatformSession(this, *SessionInfo, URH_OnlineSession::GetJoinDetailDefaults(this), FRH_GenericSuccessWithErrorBlock());
+	auto AuthContext = GetAuthContext();
+	if (!AuthContext.IsValid() || !AuthContext->IsLoggedIn())
+	{
+		// we need to wait for the player to log in before joining the session
+		SetPlatformSessionToJoinOnUserChange(*SessionInfo);
```

Comment:
If they instead login as a different user on change, are we concerned they will try to join the session the other user was invited to?

Reply:
While a valid case, it is actually not a trivial solve.  It is hard to tell if the new member was the one invited, because at the time of invite (before login), we only know the controller index, but not the actual platform player id (since they may not have completed OSS login yet).
In the majority of cases where this is used (and where it is a cert blocker), the player is unable to change their login user.  If the game needs more complicated handling, they would need to disable the autojoin setting and handle it specifically.